### PR TITLE
OFS-89: changes related to "PolicyHolder" module - new custom filter

### PR DESCRIPTION
**Pull request by sniedzielski**
In that pull request I want to merge minor changes related to PolicyHolder module changes. I had to add custom filter to check if object is deleted or not.

### CODE REVIEW

File: core/models.py [hunk 1]
```diff
@@ -560,8 +560,8 @@ class HistoryModel(DirtyFieldsMixin, models.Model):
     json_ext = FallbackJSONField(db_column="Json_ext", blank=True, null=True)
     date_created = models.DateTimeField(db_column="DateCreated", null=True)
     date_updated = models.DateTimeField(db_column="DateUpdated", null=True)
-    user_created = models.ForeignKey(User, db_column="UserCreatedUUID", related_name='user_created', on_delete=models.deletion.DO_NOTHING)
-    user_updated = models.ForeignKey(User, db_column="UserUpdatedUUID", related_name='user_updated', on_delete=models.deletion.DO_NOTHING)
+    user_created = models.ForeignKey(User, db_column="UserCreatedUUID", related_name='%(class)s_user_created', on_delete=models.deletion.DO_NOTHING, null=True)
```
Comment (dragos-dobre) [inline core/models.py:563]: I think this must be mandatory. @delcroip can you confirm this?

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: core/models.py (hunk 1 above)
Comment (sniedzielski) [reply core/models.py:563]: According to the https://openimis.atlassian.net/wiki/spaces/OP/pages/1904738367/Update+of+the+core+module - this field is 'technical'.  ("Technical fields have **")
Moreover the user who creates object is assigned automatically as the value of this field ('user_created') after saving object. And on every update the value of "user_updated" is changed to the user, who updated this object. This field is updated automatically on every save/update or deletion/replacement. Also 'user_created' is updated automatically after saving newly created object.

Comment (delcroip) [reply core/models.py:563]: yes it is force by backend so why not forbinding null ?

Comment (sniedzielski) [reply core/models.py]: Ok @delcroip and @dragos-dobre - I've changed that and it works after changing it.
Ok, I have to make changes on another repo "policy holder backend" after Patrick's review.

**Review by delcroip — APPROVED** [review]

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**